### ISSUE TITLE
Which version of pd/libpd is MobMuPlat using?

### ISSUE THREAD

**Issue by blindmanonacid** [monkeyswarm/MobMuPlat#81]
And are there any plans to compile with the latest version? It would be great!

Also thank you so much for developing this. I've been searching for a phone app that can play sounds triggered by OSC from another device, for use in theater, and this is the only thing that I can see doing it. Updating libpd (and maybe adding some external libraries?) would make it even greater :)

**Comment by porres** [comment]
yeah, I also think this is outdated and cannot be sure of what version it is using, I am always using the latest Pd version, now at 0.53-1
